nestkernel/modelrange_manager.cpp

```diff
@@ -112,24 +112,7 @@ ModelRangeManager::get_model_id( index node_id ) const
 nest::Model*
 nest::ModelRangeManager::get_model_of_node_id( index node_id )
 {
-  return kernel().model_manager.get_model( get_model_id( node_id ) );
-}
-
-bool
-ModelRangeManager::model_in_use( index i ) const
```

Comment:
Why is this no longer needed?

Reply:
I don't know. IIRC it has never been used anywhere, so I removed it.